### PR TITLE
Add In-Kind sponsors blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @inovizz @raukadah

### CODE REVIEW

**Comment by inovizz** [comment]
Looks good to me, but let's wait for others to review. 

cc @sayanchowdhury @raukadah @anistark @RiteshAgrawal @palnabarun

**Review by anistark — APPROVED** [review]

**Review by vipulgupta2048 — APPROVED** [review]

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:8]: Might be better to break that first sentence. With the 2 `and` it's quite long

**Comment by inovizz** [comment]
@palnabarun - can we merge and deploy this?

@raukadah - can we plan for social media announcements after this is merged?

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:8]:
```suggestion
As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices for the conference have been kept very minimal so everyone could afford them. Henceforth, this year we initiated In-kind Sponsorships at PyCon India. Companies and organizations with an interest to contribute and add value to event and the community can come forward to make the most out of this opportunity.
```

Better alternatives/suggestions for initiating needed.

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:12]:
```suggestion
In-kind sponsorship is a type of sponsorship where the sponsor agrees to provide swags (either physical or virtual, value-in-kind) instead of cash as part of the sponsorship agreement with PyCon India. These swags or benefits would be made directly available to all PyCon India 2021 attendees. In return for this value-added sponsorship, PyCon India 2021 team would be providing certain benefits to sponsors before, during, and after the event. 
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:16]:
```suggestion
This act of goodwill while giving back to the community helps sponsors increase their brand exposure and elevate their business identity within the Python community. Sponsors also get to have improved interaction with the PyCon India attendees and get a chance to be represented at the forefront of the biggest Python community in India. The sponsors will additionally be provided with:
```

Getting the tone right.

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:18]:
```suggestion
* Company logo on the website and PyCon India 2021 program guide.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:19]:
```suggestion
* One social media post on all official platforms and blog announcement.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:20]:
```suggestion
* Mention in the opening and closing notes during the conference (18th - 19th Sep)
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:21]:
```suggestion
* Logo display at the end of the talk.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:22]:
```suggestion
* One email to all registered attendees.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:23]:
```suggestion
* Sponsors listing on the PyCon India conference platform - Hopin.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:24]:
```suggestion
* One free ticket to each sponsor.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:25]:
```suggestion
* Assistance from PyCon India team to send digital swags to attendees.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [reply content/blog/2021/announcing-in-kind-sponsorship.md:12]: Yo @anistark we doing physical too? 
if not @Sc0rpi0n101 disregard that bit

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:29]:
```suggestion
With respect to this year's conference being hosted online. We want our prospective sponsors to follow certain rules before deciding on swags for PyCon India 2021 attendees. Please read the guidelines below carefully. 
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:34]:
```suggestion
4. If the sponsors are sending tangible swag items then logistics would have to be handled by the sponsors themselves. PyCon India team would provide necessary assistance to the sponsors if any further information is required.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:35]:
```suggestion
5. Sponsors would be allowed to send swags/goodies to only those attendees who have provided their consent and shared their address for delivery.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:36]:
```suggestion
6. Digital swags shall be shared with attendees only through PyCon India 2021 official channels. Sponsors need to forward their digital swag to the team beforehand, and the team will be sending the swags to the attendees.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:33]:
```suggestion
3. The sponsors are allowed to send brochures or promotional content up to a maximum of 6 pages to the attendees alongside the swags they will be sending.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:32]:
```suggestion
2. The swag could be digital in nature or tangible item as well but it should be approved by the PyCon India team.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:31]:
```suggestion
1. The swag should not be a promotional item, though a valid swag can definitely contain the company's branding.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:38]:
```suggestion
**Please Note: At any point in time, the PyCon India team will not be sharing any attendee-related information or data with the sponsor(s) without the explicit consent provided to the team by the attendees.**
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:42]:
```suggestion
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:1]:
```suggestion
Title: This year, PyCon India 2021 calls on In-kind sponsors. Read all about it here!
```
Too flashy? 🎃 Let's get folks excited ..

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:5]:
```suggestion
Summary: Want to give back to the Python community and reach more people among the community than ever before? Here's your chance now to put your organization in the limelight of the Python community by sponsoring the largest Python conference of this year!
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:8]: Reading this again, I think this is an important para. 
@Sc0rpi0n101 add a line about "We got a lot of queries from sponsors this year and we wanted them to still be a part of the conf. Hence this.." I remember did reach out quite a bit interested to sponsor the conf

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]
@inovizz @anistark The word "Swag" is used way too many times. Would it better if we define it somewhere clearly right at the start. Reading this the first time raises questions on could XYZ be considered swag and in line with the rules? So clarity there would be helpful. Otherwise great write-up from your side. Thanks! Let's make some noise with this one 🎉 

@Sc0rpi0n101 fixed flow, grammatical and random capitalization errors. Suggestions are inline and approved the PR by mistake. Watch out for full stops, and tense. Please also give it a run down with an error-checking tool.

I have reformatted the last couple of lines. Feel free to swap them back.

**Comment by vipulgupta2048** [comment]
@inovizz My bad, I approved it accidentally while starting to review it. The blog is still a bit rough around the edges and could use a little more work. Will hopefully get it done by tomorrow.

**Comment by vipulgupta2048** [comment]
@Sc0rpi0n101 Hey, have you had a chance to look and work on the changes suggested above?

**Comment by anistark** [comment]
@vipulgupta2048 If @Sc0rpi0n101 is unavailable, could you take this up or assign someone else from content team? 
Been pending since a long time and we are yet to announce about in-kind sponsors. 

cc: @raukadah @RiteshAgrawal  @sayanchowdhury

**Comment by vipulgupta2048** [comment]
Hey @anistark I reached out to Nikhil, he is indeed running busy at the moment and did mention that he would get back to this. If not, will get it done EOL.

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [reply content/blog/2021/announcing-in-kind-sponsorship.md:12]: @anistark ^

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [inline content/blog/2021/announcing-in-kind-sponsorship.md:42]:
```suggestion
**If you're interested in applying to be an In-kind sponsor for [PyCon India 2021](https://in.pycon.org/2021/), please fill [this form](https://forms.gle/4tfmaAfdsHPnQ37y7) at the earliest.** Each application for in-kind sponsors will be reviewed and assessed by the PyCon India team. The PyCon India team would like to understand the motivation and background of each organization applying to be an in-kind sponsor and would be making a decision after the review process is complete. We look forward to all your applications and we can't wait to see what this partnership with the organizations brings to the event this year.
```

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (anistark) [reply content/blog/2021/announcing-in-kind-sponsorship.md:12]: Yes. Can be some goodies or books, etc that they can send directly to the participants.

**Comment by anistark** [comment]
@inovizz  Please also review this.

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (raukadah) [reply content/blog/2021/announcing-in-kind-sponsorship.md:8]: what about 'started'?

**Review by raukadah — COMMENTED** [review]
Looks good, few nit picks, Can we also resolve the older comments?

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [reply content/blog/2021/announcing-in-kind-sponsorship.md:1]: How about? PyCon India 2021 opens call for ...

File: content/blog/2021/announcing-in-kind-sponsorship.md
Comment (vipulgupta2048) [reply content/blog/2021/announcing-in-kind-sponsorship.md:12]: Cool beans

**Review by vipulgupta2048 — APPROVED** [review]

**Review by raukadah — APPROVED** [review]

**Review by vipulgupta2048 — COMMENTED** [review]
Merging this

**Pull request merged**
